YouVsVirus/Assets/Scripts/Components/Player.cs

```diff
@@ -44,15 +49,21 @@ public override void SetCondition(int condition)
 		{
 			base.SetCondition(condition);
 
-			if(condition == EXPOSED)
+			if (condition == EXPOSED)
 			{
 				LevelSettings.GetActiveEndLevelController().NotifyPlayerExposed();
 			}
-			if(condition == DEAD)
+			if (condition == DEAD)
 			{
 				LevelSettings.GetActiveEndLevelController().NotifyPlayerDied();
 			}
 		}
```

Comment:
```suggestion
		}
		
```

Reply:
integrated both changes in exitkey branch